extensions/chromium/preferences_schema.json

```diff
@@ -26,6 +26,11 @@
       ],
       "default": 0
     },
+    "enableHandToolOnLoad": {
+      "description": "Deprecated. Set cursorToolOnLoad to 1 to enable the hand tool by default.",
```

Comment:
How about using an all-caps `DEPRECATED` string instead, here and elsewhere, to make it stand out a bit more?

Reply:
SGMT.